01-introduction.Rmd

```diff
@@ -0,0 +1,204 @@
+*   [AWS console](https://aws.services.alpha.mojanalytics.xyz)
+*   [Concourse](https://concourse.services.alpha.mojanalytics.xyz)
+*   [Grafana](https://grafana.services.alpha.mojanalytics.xyz)
+*   [Kibana](https://kibana.services.alpha.mojanalytics.xyz)
+
+
+## Deploy RStudio, JupyterLab and Airflow
+
+Before using RStudio, JupyterLab and Airflow, you must first deploy them.
+
+ To deploy RStudio, JupyterLab and Airflow on the Analytical Platform, you should complete the following steps:
+
+1.  Go the Analytical Platform [control panel](https://cpanel-master.services.alpha.mojanalytics.xyz).
+2.  Select the __Analytical tools__ tab.
+3.  Select the __Deploy__ buttons next to RStudio, JupyterLab and Airflow.
+
+It may take a few minutes for the tools to deploy.
+
+## Open RStudio, JupyterLab or Airflow
+
+To open RStudio, JupyterLab or Airflow:
+
+1.  Go the Analytical Platform [control panel](https://cpanel-master.services.alpha.mojanalytics.xyz).
+2.  Select the __Analytical tools__ tab.
+3.  Select the __Open__ button to the right of the tool's name.
+
+## Restart RStudio, JupyterLab or Airflow
+
+If your RStudio, JupyterLab or Airflow is not working as expected, it may help to restart it.
+
+To restart RStudio, JupyterLab or Airflow:
+
+1.  Go the Analytical Platform [control panel](https://cpanel-master.services.alpha.mojanalytics.xyz).
+2.  Select the __Analytical tools__ tab.
+3.  Select the __Restart__ button to the right of the tool's name.
+
+## Unidle Rstudio, JupyterLab or Airflow
+
+If your RStudio, JupyterLab or Airflow instance is inactive for an extended period of time (e.g., overnight) it will be idled.
+
+1.  Go the Analytical Platform [control panel](https://cpanel-master.services.alpha.mojanalytics.xyz).
+2.  Select the __Analytical tools__ tab.
+3.  Select the __Unidle__ button to the right of the tool's name. 
+
+Unidling usually only takes a few seconds, however, it can take up to several minutes.
+
+## Configure Git and GitHub
+
+To configure Git and GitHub for the Analytical Platform, you must complete the following steps:
+
+1.  [Create an SSH key](#create-an-ssh-key).
+2.  [Add the SSH key to GitHub](#add-the-ssh-key-to-github).
+3.  [Configure your username and email in Git on the Analytical Platform](#configure-your-username-and-email-in-git-on-the-analytical-platform).
+
+### Create an SSH key
+
+You can create an SSH key in RStudio or JupyterLab. We recommend that you use RStudio.
+
+#### RStudio
+
+To create an SSH key in RStudio, follow the steps below:
+
+1.  Open RStudio from the Analytical Platform control panel.
+2.  In the menu bar, select __Tools__ then __Global Options...__
+3.  In the options window, select __Git/SVN__ in the navigation menu.
+4.  Select __Create RSA key...__
+5.  Select __Create__.
+6.  Select __Close__ when the information window appears.
+7.  Select __View public key__.
+8.  Copy the SSH key to the clipboard by pressing Ctrl+C on Windows or ⌘C on Mac.
```

Comment:
```suggestion
8.  Copy the SSH public key to the clipboard by pressing Ctrl+C on Windows or ⌘C on Mac.
```

Reply:
We just say SSH key everywhere else, so we should probably either change all instances or none. I don't think not specifying that it is a public key will cause confusion or difficulty for users so I'd be inclined to leave as is.